Add symbolic link

Hoeze opened the pull request:
Supersedes #172

## Tasks

- [x] Request reviews from the relevant people
- [x] Update [CHANGELOG.md](https://github.com/BfArM-MVH/GRZ_QC_Workflow/blob/main/CHANGELOG.md)

kubranarci committed:
add symbolic link to final report

kubranarci committed:
changelog

# Conflicts:
#	CHANGELOG.md

kubranarci committed:
name alias as report

Hoeze committed:
use create_alias as alias path directly

Hoeze committed:
update changelog

kubranarci reviewed (approved):
this is older version of what I pushed. did you revert something?

Hoeze commented:
@kubranarci I separate the MultiQC fixes into #173.
Also, I changed the report symlinking logic to use `$create_alias` as the alias path:
https://github.com/BfArM-MVH/GRZ_QC_Workflow/blob/2f1d848ace47bb93670573eb97fa08d3527141ae/modules/local/merge_reports/main.nf#L20
https://github.com/BfArM-MVH/GRZ_QC_Workflow/blob/2f1d848ace47bb93670573eb97fa08d3527141ae/modules/local/merge_reports/main.nf#L27

---
EDIT: Just checked it again and fixed a bunch of nonsense mistakes...

Hoeze committed:
fix volatility errors

Hoeze commented:
Thanks, @kubranarci!
